### ISSUE TITLE
PreAnalyzeFields processor throwing class not found

### ISSUE THREAD

**Issue by nsoft** [nsoft/jesterj#144]
```
Caused by: java.lang.ClassNotFoundException: org.objenesis.Objenesis
	at com.needhamsoftware.unojar.JarClassLoader.findClass(JarClassLoader.java:684)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at com.needhamsoftware.unojar.JarClassLoader.loadClass(JarClassLoader.java:596)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 16 more

```

**Comment by nsoft** [comment]
doesn't show up in tests because they inherit that class from EasyMock's dependencies

**Comment by nsoft** [comment]
Fixed on JDK 11 branch but see #116 for requirements to get that going.
